### PR TITLE
Upgrade fluent bit version 3.1.9

### PR DESCRIPTION
**Pull request by gunnalahema**
Upgrade fluent-bit version to 3.1.9 , to resolve the vulnerabilities with trivy image [cf-registry.nr-ops.net/maas/docker-atlantis-fluent-bit](http://cf-registry.nr-ops.net/maas/docker-atlantis-fluent-bit)

### CODE REVIEW

**Review by luckslovez — APPROVED** [review]
💯

File: .github/workflows/run_e2e_tests.yml [hunk 1]
```diff
@@ -99,7 +99,7 @@ jobs:
 
   tear_down_test_executor_instances:
     name: Tear down test executor instances
-    if: always()
+    #if: always() #Uncomment after 3.1.9 version release
```
Comment (voorepreethi) [inline .github/workflows/run_e2e_tests.yml:102]: Why are we uncommenting this in 3.1.9

File: versions/sles_15.2.ymlx [hunk 2]
```diff
@@ -0,0 +1,5 @@
+osDistro: sles
+osVersion: 15.2
+packages:
+  - arch: x86_64
```
Comment (voorepreethi) [inline versions/sles_15.2.ymlx:4]: Can we get this confirmed before publishing to stage as well

File: .github/workflows/run_e2e_tests.yml (hunk 1 above)
Comment (jgals) [reply .github/workflows/run_e2e_tests.yml:102]: We were dealing with flaky tests timing out and giving spurious failures, which needs to be fixed post release.

File: versions/sles_15.2.ymlx (hunk 2 above)
Comment (gunnalahema) [reply versions/sles_15.2.ymlx:4]: we are not including 15.2 sles for now . should look into it later finding the right ami.

**Review by voorepreethi — APPROVED** [review]

File: .github/workflows/prerelease.yml [hunk 3]
```diff
@@ -57,7 +57,7 @@ jobs:
           sudo apt-get install -y debsigs
           bash ./scripts/sign.sh
 
-      - uses: actions/upload-artifact@v2
+      - uses: actions/upload-artifact@v3
```
Comment (maya-jha) [inline .github/workflows/prerelease.yml:60]: why this needed to be changed?

File: versions/sles_15.2.ymlx (hunk 2 above)
Comment (maya-jha) [reply versions/sles_15.2.ymlx:4]: If infra agent still supports it then which artifact they would get if they are on this OS Distro?

File: versions/windows-server-2022.yml [hunk 4]
```diff
@@ -2,6 +2,6 @@ osDistro: windows-server
 osVersion: 2022
 packages:
   - arch: win64
-    ami: ami-094aa6728b151e05a
+    ami: ami-0c808db6baea2d0ed
   - arch: win32
-    ami: ami-094aa6728b151e05a
+    ami: ami-0c25be66a8d4215e5
```
Comment (maya-jha) [inline versions/windows-server-2022.yml:7]: Why so many AMIs needed to be updated? Did they get updated recently?

File: .github/workflows/prerelease.yml (hunk 3 above)
Comment (gunnalahema) [reply .github/workflows/prerelease.yml:60]: previous artifact is deprecated .

File: versions/windows-server-2022.yml (hunk 4 above)
Comment (gunnalahema) [reply versions/windows-server-2022.yml:7]: Existing AMI's had no images , so thats why updated them.

File: versions/sles_15.2.ymlx (hunk 2 above)
Comment (gunnalahema) [reply versions/sles_15.2.ymlx:4]: they will get the previous fluent bit version which supports this OS Distro , in this case that could be 3.1.2

**Pull request merged**